tests/js/fixtures/partial/searchindex.js

```diff
@@ -0,0 +1 @@
+Search.setIndex({"alltitles": {"sphinx_utils module": [[0, "sphinx-utils-module"]]}, "docnames": ["index"], "envversion": {"sphinx": 61, "sphinx.domains.c": 3, "sphinx.domains.changeset": 1, "sphinx.domains.citation": 1, "sphinx.domains.cpp": 9, "sphinx.domains.index": 1, "sphinx.domains.javascript": 3, "sphinx.domains.math": 2, "sphinx.domains.python": 4, "sphinx.domains.rst": 2, "sphinx.domains.std": 2}, "filenames": ["index.rst"], "indexentries": {}, "objects": {}, "objnames": {}, "objtypes": {}, "terms": {"also": 0, "ar": 0, "built": 0, "confirm": 0, "document": 0, "function": 0, "html": 0, "i": 0, "includ": 0, "input": 0, "javascript": 0, "known": 0, "match": 0, "partial": 0, "possibl": 0, "prefix": 0, "project": 0, "provid": 0, "restructuredtext": 0, "sampl": 0, "search": 0, "should": 0, "thi": 0, "titl": 0, "us": 0, "when": 0}, "titles": ["sphinx_utils module"], "titleterms": {"modul": 0, "sphinx_util": 0}})
```

Comment:
I don't remember whether I told/asked you or not, but do we need a blank line at the end of this file (and the others)?

![image](https://github.com/sphinx-doc/sphinx/assets/10796600/07718a55-021a-4570-ae12-b8abd45bc072)

Reply:
As output by Sphinx itself, the `searchindex.js` file does not include an end-of-file newline, and since the fixtures are generated by Sphinx itself, I'd prefer that they don't contain those either.

At one point I did dabble with adding auto-reformatting of the fixtures -- arguably it could help when reviewing diffs that affect the index structure -- but on balance I preferred the simplicity of not adding any post-processing steps (and dependencies) for the JS file (adding some emphasis there: the index file is JavaScript code, not JSON data).